specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Unit
+
+The Pricing Unit represents a provider specified pricing measurement unit, which forms the foundation for determining unit prices of utilizing a particular resource and/or service. Common examples include the number of hours a compute appliance was running for a given period, the number of gigabytes * the number of hours (gigabyte-hours) for a storage appliance, or a cumulative count of requests for a network appliance or API service. For certain services providers can choose block pricing i.e., million requests as a base unit.
+PricingUnit is complementary to and describes how [QuantityInPricingUnit](#quantityinpricingunit) is measured.
+
+The PricingUnit column MUST be present in the billing data. This column MUST be of type String. It MUST NOT contain null if [QuantityInPricingUnit](#quantityinpricingunit) is not null. The PricingUnit value MUST match what is shown on the invoice if a pricing unit is present on the invoice.
```

Comment:
Should we mention the difference between usage unit and pricing? Im not suggesting we do but want to make sure that was discussed and agreed on.

Reply:
Would that be an appendix? (Asking not stating 🙂)

Reply:
TL;DR
If we decide to keep comments regarding PricingUnit in UsageUnit spec then we should do the other way around here as well.

I think as PricingUnit is complementary we literally had to put this information there. For considerations and relations to non-complementary units we may want to stick to appendix BUT as the UsageUnit contributors are commenting on PricingUnit we can add those comments.